app/src/app/shared/components/footer/footer.component.css

```diff
@@ -0,0 +1,11 @@
+#container {
+  align-content: center;
+  display: grid;
+  height: 5em;
+  justify-content: center;
```

Comment:
- [ ] Use `place-items: center` instead of `align-items` and `justify-content` combo